configure.ac

```diff
@@ -526,13 +518,17 @@ PRETTY_ARG_VAR([WARNING_CFLAGS], [C compiler warning flags],
 
 # Checks for library functions
 # ----------------------------
-AC_CHECK_FUNCS(fnmatch,[],[
+AC_CHECK_FUNCS(fnmatch,[have_fnmatch=yes],[
 AC_LIBOBJ([fnmatch])
-CPPFLAGS="$CPPFLAGS -I${srcdir}/fnmatch"
-AC_DEFINE(HAVE_FNMATCH)
-])
+have_fnmatch=no])
 AC_CHECK_HEADERS(fnmatch.h)
 
+AM_CONDITIONAL([HAVE_FNMATCH], [test "x$have_fnmatch" = "xyes"])
+if test "xno" = "x$have_fnmatch"; then
+    CPPFLAGS="$CPPFLAGS -I${srcdir}/fnmatch"
```

Comment:
Many **_CPPFLAGS" in Makefile.in are gone in Makefile.am.
I have to revive them.